sdk/include/ndk/rtltypes.h

```diff
@@ -1234,6 +1236,8 @@ typedef struct _RTL_UNLOAD_EVENT_TRACE
     WCHAR ImageName[32];
 } RTL_UNLOAD_EVENT_TRACE, *PRTL_UNLOAD_EVENT_TRACE;
 
+RTL_UNLOAD_EVENT_TRACE RtlpUnloadEventTrace[RTL_UNLOAD_EVENT_TRACE_NUMBER];
```

Comment:
This doesn't belong here, since it's not exported.

Reply:
OK. Fixed.